android/src/main/java/com/instabug/instabugflutter/InstabugFlutterPlugin.java

```diff
@@ -102,19 +98,18 @@ public void onMethodCall(MethodCall call, Result result) {
     /**
      * starts the SDK
      *
-     * @param application      the application Object
      * @param token            token The token that identifies the app, you can find
      *                         it on your dashboard.
      * @param invocationEvents invocationEvents The events that invoke
      *                         the SDK's UI.
      */
-    public void start(Application application, String token, ArrayList<String> invocationEvents) {
+    public void startWithToken( String token, ArrayList<String> invocationEvents) {
```

Comment:
I think we should deprecate this method and create a new `startWithToken` Java to make it easier to adopt the new version without having to migrate immediately to the new way of integrating the SDK.